contributors/guide/reviewer-guidelines.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Code Review Guidelines"
+weight: 1
+slug: "reviewer-guidelines" 
+---
+
+*This Section is new and in progress. Expect this document to change often*
+
+**Table Of Contents**
+
+- [Reviewer Guidelines](#reviewer-guidelines)
+    + [Reviewing Process](#reviewing-process)
+    + [Reviewing Ideology](#reviewing-ideology)
+    + [Something To Avoid](#something-to-avoid)
+    + [Additional Readings](#additional-readings)
+
+## Reviewing Process
+
+These are some general guidelines for review, based on the size or type of pull request. 
+
+Note that each reviewer has their own style, adapted over time, so don't feel you need to explicitly commit to every one of these guidelines.
+
+### Flakes/Test Fixes
```

Comment:
I strongly second removing this section.

Reply:
Another set of comments here marked resolved without comment or change.  Was perhaps a commit not pushed up onto the stack in the PR?